src/plugin.js

```diff
@@ -969,11 +1015,11 @@ videojs.HlsHandler.prototype.playlistUriToUrl = function(segmentRelativeUrl) {
  *  * `bytesReceived` - amount of bytes downloaded
  * `bandwidth` is the only required property.
  */
-videojs.HlsHandler.prototype.setBandwidth = function(xhr) {
+HlsHandler.prototype.setBandwidth = function(localxhr) {
```

Comment:
We capitalize the first letter of multi-word variable names (and we don't capitalize the whole acronym) so this would be `localXhr`.